video.py

```diff
@@ -0,0 +1,170 @@
+from datetime import datetime
+from customer import print_stars, get_main_menu_choice
+
+# Helper function
+def is_int(value):
+    try:
+        return int(value)
+    except ValueError:
+        return None
+
+def is_datetime(value):
+    try:
+        return datetime.strptime(value, "%m/%d/%Y")
+    except ValueError:
+        return None
+
+# Helper function for video option 3: Select a video
+def select_a_video(cvr):
+    select_by = input("Would you like to select by title or id? ").lower()
+    while select_by != "title" and select_by != "id" :
+        select_by = input("Could not select. Please select an option by entering title or id: ").lower()
+
+    if select_by=="title":
+        title = input("Which video title would you like to select? ")
+        cvr.get_video(title=title)
+        if not cvr.selected_video:
+            return "Invalid title, could not select." 
+    elif select_by=="id":
+        id = input("Which video id would you like to select? ")
+        if id.isnumeric():
+            id = int(id)
+            cvr.get_video(id=id)
+        if not cvr.selected_video:
+            return "Invalid id, could not select." 
+    
+    return cvr.selected_video
+
+# Select video option
+def get_video_choice():
```

Comment:
This is another place where the functionality is very similar between video & customers.  I'd consider writing a "menu" function that takes 'video' or 'customer' as an argument and generates this menu.